docs/ssm-on-aws/ssm-on-aws.md

```diff
@@ -0,0 +1,374 @@
+            {
+                "name": "http-response-time",
+                "type": "probe",
+                "provider": {
+                    "type": "python",
+                    "module": "chaoscloud.probes.http",
+                    "func": "time_http_call",
+                    "arguments": {
+                        "url": "${endpoint_url}"
+                    }
+                },
+                "tolerance": {
+                    "name": "http-response-time-under",
+                    "type": "probe",
+                    "provider": {
+                        "type": "python",
+                        "module": "chaoscloud.tolerances.http",
+                        "func": "response_time_under",
+                        "arguments": {
+                            "duration": 6000
+                        }
+                    }
+                }
+            }
+        ]
+    },
+    "method": [
+        {
+            "type": "action",
+            "name": "inject-cpu-load",
+            "provider": {
+                "type": "process",
+                "path": "aws",
+                "arguments": "ssm send-command --document-name \"cpu-stress\"
+--document-version \"1\" --targets '[{\"Key\":\"InstanceIds\",
+\"Values\":[\"${instance_id}\"]}]' --parameters '{\"duration\":[\"180\"],
+\"cpu\":[\"0\"]}' --timeout-seconds 60 --max-concurrency \"50\"
+--max-errors \"0\" --output-s3-bucket-name \"gtf-chaos-output\"
+--cloud-watch-output-config '{\"CloudWatchOutputEnabled\":true}'
+--region eu-west-1 --query 'Command.CommandId' > CommandId.info"
+            }
+        }
+    ],
+    "rollbacks": [
+        {
+            "type": "action",
+            "name": "Stop SSM run command",
+            "provider": {
+                "type": "process",
+                "path": "aws",
+                "arguments": "ssm cancel-command --command-id $(cat CommandId.info) "
+            },
+            "pauses": {
+                "after": 10
+            }
+        }
+    ],
+    "extensions": [
+        {
+            "name": "chaosiq",
+            "experiment_id": "461d862b-0a5f-45cf-ab66-bc0aef02a59c",
+            "objective_id": "997da1a4-c511-4337-9652-4b9a25c5cbfe",
+            "verification": {
+                "id": "dd1e52d0-454a-40b8-8864-5d34834191e6",
+                "frequency-of-measurement": 10,
+                "duration-of-conditions": 300.0
+            },
+            "team_id": "f3a36d51-2486-4eb1-9fe3-29b4ea8fb387",
+            "org_id": "ae4458ef-a457-4bd5-8e6b-7c7a21db6d7f"
+        }
+    ]
+}
+
+```
+
+The verification be run with:
+
+```bash
+ ENDPOINT_URL=http://ec2-52-18-8-61.eu-west-1.compute.amazonaws.com/random/4 \
+ INSTANCE_ID=i-086ccae1f2de9b012 \
+ chaos verify verification-ec2-responds-processor-load.json
+```
+
+The ```ENDPOINT_URL``` and the ```INSTANCE_ID```, should be replaced with values
+for your own EC2 instance.
+
+## Summary Timeline
+
+The verification will run for five minutes and can
```

Comment:
"and you can"

Reply:
Consider adding something along the lines of:

Great job! You've run your SSM verification and surfaced the experiment's information into your Timeline. The Timeline is the place where you begin to explore and understand What Happened(TM), exploring your verification as a planned incident to see how your system reacted and verify the many different parallel activities that occur at the same time during the verification's execution.